tests/matrix_free/dg_pbc_03.output

```diff
@@ -0,0 +1,73 @@
+
+DEAL:1d:boundary:uniform ::Active cells: 64
+DEAL:1d:boundary:uniform ::Cell batches: 64
+DEAL:1d:boundary:uniform ::Interior faces: 0 63 
+DEAL:1d:boundary:uniform ::Exterior faces: 63 0 
+DEAL:1d:boundary:uniform ::Boundary faces: 1 1 
+DEAL::
+DEAL:1d:periodic:uniform ::Active cells: 64
+DEAL:1d:periodic:uniform ::Cell batches: 64
+DEAL:1d:periodic:uniform ::Interior faces: 1 64 
+DEAL:1d:periodic:uniform ::Exterior faces: 64 1 
```

Comment:
This is the bug we were discussing in #18755: With 64 cells and periodicity, we should have 64 + 0 faces in each of the two lines (or 63 + 1, that does not really matter), but not 64 + 1.